dmt/main/models.py

```diff
@@ -84,6 +84,15 @@ def recent_active_projects(self):
             'AND a.completed < %s', [self.user.id, start, now])
         return sorted(set(list(projects)), key=lambda x: x.name.lower())
 
+    def resolved_items_for_interval(self, start, end):
+        # TODO - can this query return Items instead of Comments?
```

Comment:
I'm not familiar with the pmt's object model, offhand. In general, you could...
* query against Items and filter based on the comments properties, something like Item.objects.filter(comment__x=a, comment__y=z)
* filter based on the returned comments: Item.objects.filter(comment__in=Comment.objects.filter(...)